tensorflow/compiler/tf2tensorrt/utils/trt_logger.h

```diff
@@ -28,7 +28,7 @@ namespace tensorrt {
 class Logger : public nvinfer1::ILogger {
  public:
   Logger(string name = "DefaultLogger") : name_(name) {}
-  void log(nvinfer1::ILogger::Severity severity, const char* msg) override;
+  void log(nvinfer1::ILogger::Severity severity, const char* msg) noexcept override;
```

Comment:
Similar, need to add noexcept to the implementation of log in the cc file.

Reply:
Thanks, added